setup.py

```diff
@@ -153,7 +160,7 @@ def run(self):
     'build_usage': setup_docs.build_usage,
     'build_man': setup_docs.build_man,
     'sdist': Sdist,
-    'clean': Clean,
+    'clean2': Clean,
```

Comment:
looks like this is the simplest way to not shadow the setuptools/distutils "clean" command.

not very nice to use as the user now has to invoke clean and clean2...